datafusion/optimizer/src/eliminate_limit.rs

```diff
@@ -40,27 +40,45 @@ impl OptimizerRule for EliminateLimit {
     fn optimize(
         &self,
         plan: &LogicalPlan,
-        optimizer_config: &mut OptimizerConfig,
+        _optimizer_config: &mut OptimizerConfig,
```

Comment:
I find it strange to use a variable named `_something` -- I thought the Rust convention was to name variables starting with `_` when they were *not* used 🤔

Reply:
resolved.
BTW, in the followup PR, I will unify them to make all of them become `optimizer_config` if possible.